core/src/main/scala/kafka/log/Log.scala

```diff
@@ -17,37 +17,6 @@
 
 package kafka.log
 
-import java.io.{File, IOException}
```

Comment:
you shouldn't remove all these imports

Reply:
I think I might have forked from a non compiling state ...
Because I compile just fine on the main versions trunk branch.

Reply:
Oh, I forked the trunk branch :-(

Reply:
I will close this pull request and create a new one.